### DISCUSSION TITLE
write queue is full

### DISCUSSION THREAD

**Discussion by djgha2024** [bluenviron/mediamtx#3717] [Questions]
### Question

Hi

at many days i have 3-5 times "write queue is full" between 1-2 hours.
after restart the service is OK. after 2-4 restart its ok up to the next day.

there are 86 rtsp stream configured and around 60 24/7 are in use over rtsp

It is not possible to say that it always affects specific streams.
it not at the same time at this days

today i increase writeQueueSize to 2048

any tips more to locate/fix this?

greetings

**Comment by aler9** [comment]
Tips for increasing stream robustness with RTSP are in the [README](https://github.com/bluenviron/mediamtx?tab=readme-ov-file#corrupted-frames). Unfortunately you probably have to either decrease the bitrate or upgrade the network bandwidth.
